Doc/library/functions.rst

```diff
@@ -982,6 +978,10 @@ are always available.  They are listed here in alphabetical order.
       See the :ref:`integer string conversion length limitation
       <int_max_str_digits>` documentation.
 
+   .. deprecated-removed:: 3.11 3.14
```

Comment:
It should be `versionchanged`. `deprecated` should be used for the deprecation note and now removed.

Reply:
I think that `deprecated:: 3.11` should be used instead of `versionchanged:: 3.11` when the deprecation started. Now it is removed, so all is fine.

Reply:
Thanks; that makes sense. I wasn't sure whether we're supposed to remove older `deprecated` notes when things were finally removed.